src/measure_theory/measure/measure_space.lean

```diff
@@ -1841,12 +1841,21 @@ by simp only [count, sum_apply, hs, dirac_apply', ← tsum_subtype s 1, pi.one_a
 @[simp] lemma count_empty : count (∅ : set α) = 0 :=
 by rw [count_apply measurable_set.empty, tsum_empty]
 
-@[simp] lemma count_apply_finset [measurable_singleton_class α] (s : finset α) :
+lemma count_apply_finset' {s : finset α} (s_mble : measurable_set (s : set α)) :
```

Comment:
Why didn't you put a simp attribute for that one?

Reply:
This is essentially just because of forgetting, sorry!

I was hesitating a bit with the simp attributes of the new lemmas, though; basically because I think it would be bad if any of them fired in a case when some of the original ones apply (the original ones have type class assumptions and should therefore be more automatic). But I think if mathlib still compiles as it is with the simp attributes in the new lemmas, then this is probably not an issue.

I will add the simp tag.

Reply:
bors r+